lms/static/js/bookmarks/views/bookmarks_results.js

```diff
@@ -0,0 +1,80 @@
+;(function (define, undefined) {
+    'use strict';
+    define(['gettext', 'jquery', 'underscore', 'backbone'],
+        function (gettext, $, _, Backbone) {
+
+        var BookmarksResultsView = Backbone.View.extend({
+
+            el: '#courseware-search-results',
+            contentElement: '#course-content',
+
+            'error': '<i class="fa fa-exclamation-triangle message-error" aria-hidden="true"></i><span class="sr">' + gettext("Error") + '</span>',
+            'loading': '<i class="fa fa-spinner fa-pulse message-in-progress" aria-hidden="true"></i><span class="sr">' + gettext("Loading") + '</span>',
+
+            'errorMessage': gettext('An error has occurred. Please try again.'),
+            'loadingMessage': gettext('Loading'),
+
+            initialize: function () {
+                this.template = _.template($('#bookmarks_results-tpl').text());
+                _.bindAll(this, 'render');
+            },
+
+            render: function () {
+                this.$el.html(this.template({
+                    bookmarks: this.collection.models
+                }));
+                return this;
+            },
+
+            bookmarksShown: function () {
+                return this.$("#my-bookmarks").is(":visible");
```

Comment:
Below you are doing `this.$el.show()` and `this.$el.hide()` so doesn't it more sense to check the visibility of `this.$el`?

Also `isVisible` would be a better name.